.github/workflows/fixturegen.yml

```diff
@@ -22,13 +22,7 @@ jobs:
       - uses: actions/checkout@v1
       - name: Select Xcode
         run: sudo xcode-select -switch /Applications/Xcode_${{ matrix.xcode }}.app
-      - name: Build Package
-        working-directory: ./tools/fixturegen
-        run: swift build
       - name: Generate Fixture
-        working-directory: ./tools/fixturegen
-        run: swift run fixturegen --projects 1 --targets 1 --sources 1
-      - name: Build Tuist
-        run: swift build
+        run: ./fourier fixture --projects 1 --targets 1 --sources 1
       - name: Generate Fixture Project
-        run: swift run tuist generate --path ./tools/fixturegen/Fixture
+        run: ./fourier tuist generate --path Fixture
```

Comment:
Does Fourier internally fix up the path? if not it may need to be 

`--path projects/fixturegen/Fixture`

Reply:
I've adjusted it to generate it relative to the root.